tests/test_upload_task.py

```
@@ -24,462 +22,123 @@
-        result = start_upload(files, "[[:File:Main.svg]]", "user", "pass")
-        
-        assert result["done"] == 1
-        assert result["not_done"] == 0
-        assert len(result["errors"]) == 0
-    
     @patch("src.web.upload_task.mwclient.Site")
-    @patch("src.web.upload_task.upload_file")
-    def test_upload_multiple_files_success(self, mock_upload_file, mock_site_class, sample_files_to_upload):
-        """Test uploading multiple files successfully."""
+    @patch("src.web.upload_task.OAuthAuthentication")
+    def test_successful_upload_flow(
```

Pylint (4.0.1)

[refactor] Too many arguments (6/5)

(R0913)